packages/node_modules/@webex/plugin-logger/README.md

```diff
@@ -6,8 +6,10 @@
 
 - [Install](#install)
 - [Usage](#usage)
-- [Contribute](#contribute)
+  - [Levels](#levels)
```

Comment:
I know this is in the context of the logger, but just looking at the word "levels" has me thinking for half a sec, _"what levels?"_. Just to avoid the minor cognitive load, I'd just call them "log levels"

```suggestion
  - [Log Levels](#log-levels)
```